src/variables/integration/dropdown.spec.js

```diff
@@ -0,0 +1,20 @@
+describe("test dropdown menu", () => {
+  it("should open dropdown menu", () => {
+    cy.visit("localhost:3000");
+    cy.get(".gc-dropdown");
+    cy.get(".gc-dropdown__btn").click();
+  });
+  it("should choose value red", () => {
+    cy.get(".gc-dropdown__item")
+      .contains("Red")
+      .click();
+    cy.get(".gc-dropdown__btn").contains("Red");
```

Comment:
For assertions rather use explicit method `.should('contain', 'value')` - check difference https://stackoverflow.com/questions/50042670/cypress-test-is-contains-equivalent-to-shouldcontain